external/nrf52x-base/sdk/nrf5_sdk_16.0.0/integration/nrfx/legacy/nrf_drv_twi.h

```diff
@@ -161,9 +161,9 @@ typedef struct
  */
 #define NRF_DRV_TWI_DEFAULT_CONFIG                                               \
 {                                                                                \
-    .frequency          = (nrf_drv_twi_frequency_t)TWI_DEFAULT_CONFIG_FREQUENCY, \
     .scl                = 31,                                                    \
     .sda                = 31,                                                    \
+    .frequency          = (nrf_drv_twi_frequency_t)TWI_DEFAULT_CONFIG_FREQUENCY, \

```

Comment:
yea the C++ requires that configs are declared in initialization order, I think the change is probably fine

Reply:
lmao